setup.py

```diff
@@ -15,7 +15,7 @@
     "sortedcontainers<3",
     "numexpr<3",
     "dpath<3",
-    "psutil<6",
+    "psutil>=6,<7",
```

Comment:
should we be consistent about `>=lower,<bound` or just do `<bound` like below?

Reply:
I think it depends on the package itself, but I don't think consistency on this is a high priority in my opinion.